src/features/events_list/atoms/autoSelectEvent.ts

```diff
@@ -29,13 +29,17 @@ export const autoSelectEvent = createAtom(
         if (currentEvent?.id) {
           // This case happens when call for event by provided eventId didn't return event
           schedule((dispatch) =>
-            dispatch(
+            dispatch([
               currentNotificationAtom.showNotification(
                 'warning',
                 { title: i18n.t('event_list.no_event_in_feed') },
                 5,
               ),
-            ),
+              // select top disaster from the list, if it's not empty
+              currentEventAtom.setCurrentEventId(
+                (eventListResource.data || [])[0]?.eventId || null,
```

Comment:
This seems like half fix for https://kontur.fibery.io/Tasks/Task/Select-a-top-disaster-after-switching-feed-on-DN2-17741

It selects top disaster from the new feed, but the warning toast still appears. This needs to be looked into.

It doesn't fix  https://kontur.fibery.io/Tasks/Task/Deselect-a-disaster-on-DN2-after-using-map-view-filter-17740 at all